docs/en/operations/storing-data.md

```diff
@@ -113,3 +115,48 @@ Example of disk configuration:
     </storage_configuration>
 </yandex>
 ```
+
+## Storing Data on Web Server {#storing-data-on-webserver}
+
+You can store data on a web server as static files (for example, table's data directory) using a disk with type `web` and run queries on this data. It can be useful for serving public datasets.
+
+The following types of queries are not supported: [CREATE TABLE](../sql-reference/statements/create/table.md), [ALTER TABLE](../sql-reference/statements/alter/index.md), [RENAME TABLE](../sql-reference/statements/rename.md#misc_operations-rename_table), [DETACH TABLE](../sql-reference/statements/detach.md) and [TRUNCATE TABLE](../sql-reference/statements/truncate.md).
+
+Web server storage is supported only for the [MergeTree](../engines/table-engines/mergetree-family/mergetree.md) and [Log](../engines/table-engines/log-family/log.md) engine families. To access the data stored on a `web` disk, use the [storage_policy](../engines/table-engines/mergetree-family/mergetree.md#terms) setting when executing the query. For example, `ATTACH TABLE table_web UUID '{}' (id Int32) ENGINE = MergeTree() ORDER BY id SETTINGS storage_policy = 'web'`.
+
+Example of disk configuration:
```

Comment:
```suggestion
A ready test case. You need to add this configuration to config:
<xml from below>
and then execute this query 
``` <paste it https://github.com/ClickHouse/ClickHouse/blob/master/src/Disks/DiskWebServer.h#L17>